telegram/callbackquery.py

```diff
@@ -172,7 +178,8 @@ def edit_message_caption(self, caption, *args, **kwargs):
                                                  message_id=self.message.message_id,
                                                  *args, **kwargs)
 
-    def edit_message_reply_markup(self, reply_markup, *args, **kwargs):
+    def edit_message_reply_markup(self, reply_markup: 'InlineKeyboardMarkup', *args: Any,
```

Comment:
Tbh I'd just add black to pre-commit.yml aswell if every single file is being changed anyway...

```
-   repo: https://github.com/psf/black
    rev: 20.8b1
    hooks:
    -   id: black
```

Reply:
I think code formatting is worth its own discussion and PR